README.md

```diff
@@ -13,6 +13,8 @@ This library contains a collection of helpers, models and extension methods that
 - [Consistent Api Error schema.](#Sample1)
```

Comment:
Wiki needs some more updates.
- Update to include problem JSON
- Remove FluentValidation stuff
- Some JSON exception stuff needs to be removed

Reply:
This is slated for milestone 1.1 as mentioned in #8